### PR TITLE
Mudic

### PR DESCRIPTION
**Pull request by diehlpk**
Add a new loader for the CSV format generated by muDIC. There is a new attribute for the `Data ` object to specify the type either `mudic` or `vic3d`.

```
  9 Data:
  8     Dimension: 2
  7     Type: Plane_Stress
  6     Sigma: 6
  5     File:
  4         Name: test.csv
  3         Path: ./
  2         Type: mudic
```

### CODE REVIEW

File: peripydic/materials/elastic.py [hunk 1]
```diff
@@ -73,8 +73,6 @@ def compute_dilatation_slice(self, deck, data_solver, y, start, end):
                     self.dilatation[i] += (1. / self.Weighted_Volume[i]) * functions.w(data_solver, X, deck.influence_function) * linalgebra.norm(X) * self.e[i,n] * self.Volume_Correction[i,n] * deck.geometry.volumes[p]
 
                 if deck.dim == 2:
-                    if self.Weighted_Volume[i] == 0:
```
Comment (ilyasst) [inline peripydic/materials/elastic.py:76]: @diehlpk What was this for ? Do we not need it at all anymore ?

**Review by ilyasst — COMMENTED** [review]
Everything seems perfect, except the removal of weighted volume check per element. How did we use to get that variable before ? Did we have "volumeless elements" ? Could we clarify this part (line76 of peripydic/materials/elastic.py) before merging ?

Cheers

File: peripydic/materials/elastic.py (hunk 1 above)
Comment (diehlpk) [reply peripydic/materials/elastic.py:76]: It was in the previous commit but removed in the current one. Was some relict from debugging.

Comment (ilyasst) [reply peripydic/materials/elastic.py:76]: Alright.

**Review by ilyasst — APPROVED** [review]

**Comment by ilyasst** [comment]
Thank you !

**Pull request merged**